tests/test_syntax/inline/test_advanced_images.py

```diff
@@ -0,0 +1,99 @@
+from markdown.test_tools import TestCase
+
+
+class TestAdvancedImages(TestCase):
```

Comment:
We suppose can leave the class named "advanced" but let's just name the file `images.py`. Eventually all "image" related tests should be moved here.

Reply:
Sounds good.